niri-config/src/lib.rs

```diff
@@ -422,6 +422,8 @@ pub struct Gradient {
     pub angle: i16,
     #[knuffel(property, default)]
     pub relative_to: GradientRelativeTo,
+    #[knuffel(property, str)]
```

Comment:
```suggestion
    #[knuffel(property(name="in"), str)]
```
Something like this I think